### PR TITLE
Adding Krado66 Keyboard

### PR DESCRIPTION
**Pull request by Feags**
Adding Krado Industries keyboard Krado66 vial keymap and supporting files.

The Krado66 is already in the QMK repo with VIA keymaps. This PR adds /vial folder with config.h keymap.c rules.mk and vial.json.

### CODE REVIEW

File: keyboards/kradoindustries/krado66/keymaps/vial/config.h [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+#define DRIVER_LED_TOTAL 14
+#define RGBLIGHT_HUE_STEP 8
+#define RGBLIGHT_SAT_STEP 8
+#define RGBLIGHT_VAL_STEP 8
+#define VIALRGB_NO_DIRECT
+
+#define DYNAMIC_KEYMAP_MACRO_COUNT 6
+#define DYNAMIC_KEYMAP_LAYER_COUNT 3
+
+#define VIAL_ENCODER_DEFAULT { KC_PGDN, KC_PGUP, KC_PGDN, KC_PGUP, KC_VOLD, KC_VOLU, RGB_MOD, RGB_RMOD, KC_PGDN, KC_PGUP, KC_PGDN, KC_PGUP, KC_VOLD, KC_VOLU, KC_LEFT, KC_RGHT, KC_LBRC,  KC_RBRC, KC_LBRC, KC_RBRC, C(KC_EQL), C(KC_MINS), KC_WH_U, KC_WH_D}
```
Comment (lesshonor) [inline keyboards/kradoindustries/krado66/keymaps/vial/config.h]: Obsolete and redundant settings removed.
```suggestion
#define VIALRGB_NO_DIRECT

#define DYNAMIC_KEYMAP_MACRO_COUNT 6
#define DYNAMIC_KEYMAP_LAYER_COUNT 3
```

File: keyboards/kradoindustries/krado66/keymaps/vial/rules.mk [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+VIAL_ENABLE = yes
+VIAL_ENCODERS_ENABLE = yes
+COMBO_ENABLE = no
+QMK_SETTINGS = yes
+ENCODER_ENABLE = yes
+ENCODER_MAP_ENABLE = yes
+RGB_MATRIX_DRIVER = WS2812
+KEY_OVERRIDE_ENABLE = no
+TAP_DANCE_ENABLE = no
+MOUSEKEY_ENABLE = no
```
Comment (lesshonor) [inline keyboards/kradoindustries/krado66/keymaps/vial/rules.mk]: Obsolete and redundant rules removed.
```suggestion
COMBO_ENABLE = no
QMK_SETTINGS = yes
ENCODER_MAP_ENABLE = yes
KEY_OVERRIDE_ENABLE = no
TAP_DANCE_ENABLE = no
MOUSEKEY_ENABLE = no
```

File: keyboards/kradoindustries/krado66/keymaps/vial/vial.json [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+  "vendorId": "0x4B72",
+  "productId": "0x6664",
+  "lighting": "qmk_rgblight",
+  "menus": [
+    "via/keymap",
+    "via/layouts",
+    "via/macros",
+    "via/save_load",
+    "core/qmk_rgblight"
+  ],
```
Comment (lesshonor) [inline keyboards/kradoindustries/krado66/keymaps/vial/vial.json]: Attributes not used by Vial removed.
```suggestion
  "lighting": "qmk_rgblight",
```

**Review by lesshonor — CHANGES_REQUESTED** [review]

**Comment by Feags** [comment]
@lesshonor Suggestions committed.

**Pull request merged**